protocol/flatbuffers/rpc.fbs

```diff
@@ -142,7 +142,10 @@ table ChangeSkeletonConfigRequest {
     value: float;
 }
 
-table OpenSerialRequest {}
+table OpenSerialRequest {
+    rts: bool;
```

Comment:
Please add comments (using `///`) as to what these fields are

Reply:
```
table OpenSerialRequest {
    /// rts: Status of the Serial Line "Request to Send" datapin cached. On Dev-Boards used for RST or Flash Pin
    /// dtr: Status of the Serial Line "Data Terminal Ready" datapin cached. On Dev-Boards used for RST or Flash Pin
    rts: bool;
    dtr: bool;
}

/// SerialSetCtrlRequest
/// Changing the Serial RTS and DTR signal from the Serial Line.
/// This signals are common used on Dev-Boards (used in DIY Slimes) for the Flash- or Reset-Pin
/// Some Drivers or Boards use this Pin not the same way. So if you open the WiFi Window,
/// the Reset Pin or the Flash pin is pressed all the time and does not allow a normal Boot.
/// This Control is to give the User control over this function.
table SerialSetCtrlRequest {
    rts: bool;
    dtr: bool;
}
```

Reply:
still very confused what the funtion of this is. Are you trying to control the logic level of hardware signals from the protocol? I thought serial only has rx and tx.

Reply:
RS232 does have up to 9 pins (DSub-9) (Its "commonly" known as Com-Port)
CH340C used in this schematic as example https://www.wemos.cc/en/latest/_static/files/sch_d1_mini_v4.0.0.pdf
uses the RTS and DTR line to pull RST or IO0 over the U5 (UMH3N) to low or not.

Other Schematics:
ESP32 https://cdn.sparkfun.com/assets/learn_tutorials/5/0/7/esp32-thing-schematic.pdf
ESP8266 https://cdn.sparkfun.com/datasheets/IoT/SparkFun-Blynk-Board-ESP8266.pdf
ESP8266 https://cdn.sparkfun.com/datasheets/Wireless/WiFi/ESP8266-Thing-Dev-v10.pdf

See also "ESP to Serial" https://arduino-esp8266.readthedocs.io/en/latest/boards.html

On the "ESP8266-DevKitC-V1.0_reference_design" it seems to be not used.

Reply:
pls close if resolved for you

Reply:
Ok Ill have to read this but I don't understand it rn. Instead of blocking this PR can one other person confirm that this makes sense to merge? Maybe @TheDevMinerTV?

Reply:
I believe that this makes sense to have in the protocol since some Amazon boards don't have these wires hooked up correctly or the driver is doing stuff to them so they don't work in the server. You can override these lines on the UART connection though, which this will enable.